dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -95,57 +103,69 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
-                    err,
-                )
-            ) from err
+            raise err
 
     # Get the positional arguments (the inputs) from the pipeline_config
     # dict and run the specified sanity checker for it
     args = []
     for match, sanity_checker in pipeline_config["transform_inputs"].items():
         try:
             input_file_path: Path = local_store.save_lone_file_matching(match)
+            logger.info(f"Successfully saved file that matches pattern to path {input_file_path}", data={"input_file_path": input_file_path})
         except Exception as err:
+            logger.error("Error occured while attempting to save matching pattern file.", 
+            err, 
+            data={"match": match, 
+            "pipeline_config": pipeline_config, 
+            "local_store": local_store})    
+
             de_notifier.failure()
-            printable_transform_details = json.dumps(
-                pipeline_config, indent=2, default=lambda x: str(x)
-            )
-            raise Exception(
-                message.pipeline_input_exception(
-                    printable_transform_details, local_store, err
-                )
-            ) from err
+
+            raise err
 
         try:
             sanity_checker(input_file_path)
+            logger.info("Successfully ran sanity check on input file path.", data={"input_file_path": input_file_path})
         except Exception as err:
+            logger.error("Error occured while running sanity checker on input file path.", 
+            err, 
+            data={"input_file_path": input_file_path, 
+            "pipeline_config": pipeline_config, 
+            "local_store": local_store})
+
             de_notifier.failure()
-            raise Exception(
-                message.pipeline_input_sanity_check_exception(
-                    pipeline_config, local_store, err
-                )
-            ) from err
+
+            raise err
 
         args.append(input_file_path)
 
     # Get the transform function and keyword arguments from the transform_details
     transform_function = pipeline_config["transform"]
     kwargs = pipeline_config["transform_kwargs"]
+    logger.info("Retrieved transform function and keyword arguments from transform details.", data={"pipeline_config": pipeline_config})
 
     try:
         csv_path, metadata_path = transform_function(*args, **kwargs)
+        logger.info("Successfully retrieved csv path and metadata path with keyword args from transform function", 
+        data={"transform_function": transform_function, 
+        "pipeline_config": pipeline_config})
+
     except Exception as err:
-        de_notifier.failure()
-        printable_transform_details = json.dumps(
-            pipeline_config, indent=2, default=lambda x: str(x)
+        logger.error("Error occured while getting transform function and keyword arguments from transform details", 
```

Comment:
error when running transform, include the name of the transform, args and kwargs with the data keyword.